tests/admin-notice/test-class-admin-notice.php

```diff
@@ -17,12 +17,12 @@ public function test__display() {
 		$notice->display();
 	}
 
-	public function test__display_with_button() {
+	public function test__display_dissmissible() {
```

Comment:
Typo in test method name.

Reply:
Uh my Czenglish betrays me again. What is the correct form? When I googled I found `dismissible - subject to dismissal` which seems to be what I typed, but sometimes I am really blind on these things.

Reply:
Oh it is the single 's' now I see it:)